.github/workflows/ci.yml

```
@@ -109,6 +109,9 @@ jobs:
     timeout-minutes: 60
     steps:
       - uses: taiki-e/checkout-action@v1
+      # QUICK WORKAROUND FOR CI ISSUE with libc & MSRV
+      - run: cargo add libc@=0.2.153 --no-default-features -p futures-executor
+      - run: git clean -dfX
```

Should update something like this:

```suggestion
      - name: Workaround for CI issue in futures-executor between libc & MSRV
        run: |
          cargo add libc@=0.2.153 --no-default-features -p futures-executor
          git clean -dfX
```

similar to this:
- https://github.com/seanmonstar/num_cpus/blob/815551cfc7f9393410de47e4bd1fe77d11f6ace5/.github/workflows/ci.yml#L39-L40

It may be possible for this to work with some newer version of `libc`, may need some time for trial-and-error.

I am actually not 100% happy about this kind of workaround. IMHO it should be possible to simply install & work with MSRV.